app/src/components/Form/Form.js

```diff
@@ -4,17 +4,33 @@ import { Formik } from 'formik';
 import * as Yup from 'yup';
 import _ from 'lodash';
 
-export const formFromFields = (fields, formikProps) =>
-  Object.keys(fields).map(id =>
-    React.createElement(fields[id].component, {
+export const formFromFields = (fields, formikProps, dynamicRequire) =>
+  Object.keys(fields).map(id => {
+    return React.createElement(fields[id].component, {
       key: id,
       id,
       label: fields[id].label,
       options: { ...fields[id].options },
       formik: formikProps,
-      isRequired: _.get(fields[id], 'validation._exclusive.required'),
-    })
-  );
+      isrequired: !!(
+        dynamicRequire[id] ||
+        _.get(fields[id], 'validation._exclusive.required')
+      ),
+    });
+  });
+// export const getIsVisible = (validate, values) => {
+//   const show = validate(values);
+//   if (show._visibleIf) {
+//     return show._visibleIf;
+//   }
+//   return {};
+// };
+
+function removeVisibleIfFromErrorObject(errorObject) {
```

Comment:
replace with https://lodash.com/docs/4.17.15#omit

Reply:
Never mind -- it looks like they are removing that method in v5 and decided that people should use _.pick only. Can we pick the properties we want only rather than deleting one?

Reply:
Great, done.

Reply:
Pick works the opposite way of omit, you would actually have to pass in the properties that you wanted to *keep* -- so this needs to change